js/everything.js

```diff
@@ -427,6 +428,35 @@ function anchorURLs(text) {
     return text.replace(re, "<a href=\"$1\" target=\"_blank\">$1</a>");
 }
 
+function anchorTimestamps(text, videoID) {
+    /* RegEx to match
+      hh:mm:ss
+      h:mm:ss
+      mm:ss
+      m:ss
+    and wraps the timestamps in <a> tags
+    RegEx explanation:
+    ((?:[0-5]\d|\d|) either the string is "colon 00-59" or "0-9" or "blank"
+    (?:\d|\:[0-5]\d) either the string is "colon 0-9" or "colon 00-59"
+    (?:$|\:[0-5]\d)) either the string ends or is a a number between 00-59
+    */
+    var re = /((?:[0-5]\d|\d|)(?:\d|\:[0-5]\d)(?:$|\:[0-5]\d))/g;
+    return text.replace(re, function(match) { return "<a href=\"" + makeListenURL(videoID, convertTimestamp(match)) + "\">" + match + "</a>"; });
```

Comment:
Re-commenting to make sure we don't forget about this https://github.com/zen-audio-player/zen-audio-player.github.io/pull/234#discussion_r73083405